src/electron/process_manager.ts

```diff
@@ -280,27 +188,39 @@ export class ConnectionManager {
 //       found).
 class ChildProcessHelper {
   private process?: ChildProcess;
+  private running = false;
 
-  private exitListener?: () => void;
+  protected exitListener?: (code?: number, signal?: string) => void;
+  protected stdoutListener?: (data?: string | Buffer) => void;
 
   constructor(private path: string) {}
 
-  protected launch(args: string[]) {
+  launch(args: string[]) {
     this.process = spawn(this.path, args);
+    this.running = true;
 
-    const onExit = () => {
+    const onExit = (code?: number, signal?: string) => {
+      this.running = false;
       if (this.process) {
         this.process.removeAllListeners();
       }
       if (this.exitListener) {
-        this.exitListener();
+        this.exitListener(code, signal);
+      }
+    };
+
+    const onStdoutData = (data?: string | Buffer) => {
+      if (this.stdoutListener) {
+        this.stdoutListener(data);
       }
     };
 
     // We have to listen for both events: error means the process could not be launched and in that
     // case exit will not be invoked.
     this.process.on('error', onExit.bind((this)));
     this.process.on('exit', onExit.bind((this)));
+    this.process.stdout.on('data', onStdoutData.bind(this));
+    this.process.stderr.on('data', onStdoutData.bind(this));
```

Comment:
The process is using stderr. Not sure why this is the case, since Go ultimately calls `log.Printf`, which should output to stdout. Changed the implementation to listen on stderr only.